.github/synthesis/staging.json

```diff
@@ -1,8 +1,7 @@
 [
-  {"top": "fullMeshHwCcTest",              "stage": "test", "targets": "All"},
-  {"top": "fullMeshSwCcTest",              "stage": "test", "targets": "All"},
-  {"top": "linkConfigurationTest",         "stage": "test", "targets": "All"},
-  {"top": "safeDffSynchronizer",           "stage": "hdl"},
```

Comment:
`safeDffSynchronizer` didn't cause problems with this PR, right?

Reply:
Nope :). Accidental delete!